capstone-frontend/src/App.jsx

```diff
@@ -26,16 +27,36 @@ class App extends Component {
 
     const userID = window.localStorage.getItem("userID");
 
-    const bookLists = await fetch(`/api/booklist?userID=${userID}`, {
+    let bookLists = await fetch(`/api/booklist?userID=${userID}`, {
       method: "GET",
     }).then(resp => resp.json()).catch(err => console.log(err));
```

Comment:
Should this be broken into multiple lines to be consistent with the rest of the fetches we have in the application?

Reply:
Done.